{
 "pydata_google_auth": {
  "lineno": null,
  "symbols_in_volume": [],
  "type": "module"
 },
 "pydata_google_auth._version": {
  "lineno": null,
  "symbols_in_volume": [],
  "type": "module"
 },
 "pydata_google_auth._version.get_versions": {
  "lineno": 20,
  "symbols_in_volume": [
   "json.loads"
  ],
  "type": "function"
 },
 "pydata_google_auth.auth": {
  "lineno": null,
  "symbols_in_volume": [
   "logging.getLogger"
  ],
  "type": "module"
 },
 "pydata_google_auth.auth.default": {
  "lineno": 27,
  "symbols_in_volume": [
   "pydata_google_auth.cache.READ_WRITE",
   "pydata_google_auth.exceptions.PyDataCredentialsError"
  ],
  "type": "function"
 },
 "pydata_google_auth.auth.get_application_default_credentials": {
  "lineno": 112,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pydata_google_auth.auth.get_user_credentials": {
  "lineno": 153,
  "symbols_in_volume": [
   "google_auth_oauthlib.flow.InstalledAppFlow.from_client_config",
   "pydata_google_auth.cache.READ_WRITE",
   "pydata_google_auth.exceptions.PyDataCredentialsError"
  ],
  "type": "function"
 },
 "pydata_google_auth.cache": {
  "lineno": null,
  "symbols_in_volume": [
   "logging.getLogger"
  ],
  "type": "module"
 },
 "pydata_google_auth.cache.CredentialsCache": {
  "lineno": 113,
  "symbols_in_volume": [],
  "type": "class"
 },
 "pydata_google_auth.cache.CredentialsCache.load": {
  "lineno": 120,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pydata_google_auth.cache.CredentialsCache.save": {
  "lineno": 134,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pydata_google_auth.cache.ReadWriteCredentialsCache": {
  "lineno": 148,
  "symbols_in_volume": [],
  "type": "class"
 },
 "pydata_google_auth.cache.ReadWriteCredentialsCache.__init__": {
  "lineno": 163,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pydata_google_auth.cache.ReadWriteCredentialsCache.load": {
  "lineno": 167,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pydata_google_auth.cache.ReadWriteCredentialsCache.save": {
  "lineno": 179,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pydata_google_auth.cache.WriteOnlyCredentialsCache": {
  "lineno": 191,
  "symbols_in_volume": [],
  "type": "class"
 },
 "pydata_google_auth.cache.WriteOnlyCredentialsCache.__init__": {
  "lineno": 209,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pydata_google_auth.cache.WriteOnlyCredentialsCache.save": {
  "lineno": 213,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pydata_google_auth.cache._get_default_credentials_path": {
  "lineno": 18,
  "symbols_in_volume": [
   "os.environ",
   "os.makedirs",
   "os.name",
   "os.path.exists",
   "os.path.expanduser",
   "os.path.join"
  ],
  "type": "function"
 },
 "pydata_google_auth.cache._load_user_credentials_from_file": {
  "lineno": 64,
  "symbols_in_volume": [
   "json.load"
  ],
  "type": "function"
 },
 "pydata_google_auth.cache._load_user_credentials_from_info": {
  "lineno": 42,
  "symbols_in_volume": [],
  "type": "function"
 },
 "pydata_google_auth.cache._save_user_account_credentials": {
  "lineno": 94,
  "symbols_in_volume": [
   "json.dump"
  ],
  "type": "function"
 },
 "pydata_google_auth.exceptions": {
  "lineno": null,
  "symbols_in_volume": [],
  "type": "module"
 },
 "pydata_google_auth.exceptions.PyDataCredentialsError": {
  "lineno": 1,
  "symbols_in_volume": [],
  "type": "class"
 }
}